whois-api/src/main/java/net/ripe/db/whois/api/rest/HttpsBasicAuthFiler.java

```diff
@@ -31,14 +31,6 @@ public void doFilter(final ServletRequest request, final ServletResponse respons
             return;
         }
 
-        final HttpServletRequest httpRequest = (HttpServletRequest) request;
```

Comment:
Is this class needed?
you are already creating a filter called `HttpsAuthHeaderFiler`

Reply:
It is needed till we support passwords in basic auth header. Check the HttpBasicAuthRequestWrapper